jobs/config.json

```diff
@@ -9347,6 +9380,32 @@
       "sig-node"
     ]
   },
+  "pull-federation-e2e-gce": {
```

Comment:
does the new pr-federation job require another deploy job or not? How does it work now?

Reply:
It can work with existing deploy job. maybe for sanity we can change the name of the deploy job suitably

Reply:
In deploy job all we do is bringup k8s clusters with latest ci commits in k/k. the new pr federation job can just reuse the clusters created by this deploy job